### PR TITLE
Dockerize

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
Motivation:
Dependency management is not fun. Let docker explicitly define what it takes to get this project running.

### CODE REVIEW

**Comment by aureliosaraiva** [comment]
I created a Docker in my project, it is fork the guides.

https://github.com/ember-brasil/website

**Comment by jenweber** [comment]
**Some notes for people helping with testing:**

From the guides repo, `git remote add nullvoxpopuli https://github.com/NullVoxPopuli/guides.git`

`git checkout add-docker-for-dependency-less-dev`. 

Install Docker from https://www.docker.com/

`docker-compose build && docker-compose up`

View at http://localhost:4567/

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+version: '3'
```
Comment (aureliosaraiva) [inline docker-compose.yml]: I recommend to use version 2 the docker-compose. It improve the compatibility with various version the docker-compose. Version 2 is compatible with 1.9+.

File: Gemfile.lock [hunk 2]
```diff
@@ -244,4 +244,4 @@ DEPENDENCIES
   wdm (>= 0.1.0)
 
 BUNDLED WITH
-   1.15.1
+   1.15.4
```
Comment (aureliosaraiva) [inline Gemfile.lock]: You can defined the version bundler, thus it not break the current environment of other developers who do not use docker.

`RUN gem install bundler -v 1.15.1`

https://github.com/ember-brasil/website/blob/master/docker-images/ember-guides#L19

Comment (NullVoxPopuli) [reply Gemfile.lock]: updated!

**Comment by jenweber** [comment]
@NullVoxPopuli Can you squash your commits into one?

I'll do README updates in a separate PR

**Comment by jenweber** [comment]
@locks pending a squash of commits, any objections to merging this? I wanted it to be tested on some other machines, but it's a huge improvement even if it doesn't work for everyone.

**Comment by locks** [comment]
@jenweber seems fine, and we can even squash via GitHub, too.

**Pull request merged**